### ISSUE TITLE
publish module to npm

### ISSUE THREAD

**Issue by macneib** [seedalpha/rethink#4]
Do you intend to register/publish this module on npm?

**Comment by rusintez** [comment]
Yes, we are planning to prertify docs, increase coverage and release it to npm soon.

**Comment by macneib** [comment]
Understood. Can we keep this issue open until it's release?

**Comment by rusintez** [comment]
definitely
